pages/_app.tsx

```diff
@@ -1,8 +1,9 @@
 import '../styles/globals.css';
 import {createTelemetryClient, TelemetryProvider} from '../lib/telemetry';
 import { Provider } from 'next-auth/client';
+import type { AppProps } from "next/app"
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689300)
```suggestion
import type { AppProps } from "next/app";
```